calc_electromagnetic.py

```diff
@@ -91,14 +91,14 @@ def process(sigma, field, name):
     # -------------------------------------------
     # tabulated values of s_kin = s - mc^2
     # Note: integration method (Romberg) requires 2^n + 1 log-spaced tabulation points
-    s_kin = np.logspace(6, 23, 2049) * eV**2
+    s_kin = np.logspace(0, 23, 2 ** 18 + 1) * eV**2
     xs = getTabulatedXS(sigma, s_kin)
     rate = interactionRate.calc_rate_s(s_kin, xs, E, field)
 
     # save
     fname = folder + '/rate_%s.txt' % field.name
     data = np.c_[np.log10(E / eV), rate]
-    fmt = '%.2f\t%.9g'
+    fmt = '%.2f\t%5.4e'
```

Comment:
Could you increase the precision by a couple of digits? Otherwise the rates will have some stair features.